### DISCUSSION TITLE
Refresh Token

### DISCUSSION THREAD

**Discussion by Avinar** [codeigniter4/shield#1061] [Ideas]
Firstly, Thanks for Useful Shield authentication package. Secondly, I work with JWT Token and handle authorization. This worked charming but I think for token expiration we need to add refresh token class, config, filter, ... to handle authorization continuously with new JWT token and handle refresh token safely. Shield has a solution for refresh token idea?

**Comment by kenjis** [comment]
Currently, Shield has no mechanism to refresh tokens.

See #1083
